downloads.html

```diff
@@ -86,6 +86,8 @@ <h4>Packages</h4>
     <p>Those may be out of date at times. If they are too out of date, you can build from source.</p>
 
     <ul>
+      <li>Flatpak -
```

Comment:
For the reason that all other list points are distributions, this might fit better:
`Flatpak (any Linux) -` or `Flatpak (all distributions) -`